README.md

```diff
@@ -85,6 +85,8 @@ To run different web applications for diffent virtual hosts:
          / --httpsCertificate     file and the corresponding certificate file
        --httpsRedirectHttp      = redirect http requests to https (requires both --httpPort and --httpsPort)
        --http2Port              = set the http2 listening port. -1 to disable, Default is disabled
+       --httpsSniHostCheck     = if the SNI Host name must match when there is an SNI certificate. Check disabled per default
```

Comment:
The equals sign is not aligned with the other equals signs on the preceding line and the next line:

```suggestion
       --httpsSniHostCheck      = if the SNI Host name must match when there is an SNI certificate. Check disabled per default
```